hiera.yaml

```diff
@@ -15,5 +15,8 @@ hierarchy:
   - name: 'Distribution Name'
     path: '%{facts.os.name}.yaml'
 
+  - name: 'Operating System Family'
+    path: '%{facts.os.family}.yaml'
```

Comment:
Do we still need this?

Reply:
In one hand, module supports CentOS, RedHat and Fedora.
In second hand, before this PR, common in hiera data handle values for the RedHat family OSes.
And finally, in this PR, we remove the common from hiera data in module.

So, now, yes it is needed to handle values for RedHat family OSes.